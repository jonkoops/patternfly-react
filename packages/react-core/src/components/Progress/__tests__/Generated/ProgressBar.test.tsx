/**
 * @vitest-environment jsdom
 */
import { render } from '@testing-library/react';
import React from 'react';
import { expect, it } from 'vitest';

import { ProgressBar } from '../../ProgressBar';

it('ProgressBar should match snapshot (auto-generated)', () => {
  const { asFragment } = render(<ProgressBar children={<>ReactNode</>} className={"''"} value={42} />);
  expect(asFragment()).toMatchSnapshot();
});
