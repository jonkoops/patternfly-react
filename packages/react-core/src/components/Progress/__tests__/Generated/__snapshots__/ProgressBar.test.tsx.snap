// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ProgressBar should match snapshot (auto-generated) 1`] = `
<DocumentFragment>
  <div
    class="pf-v5-c-progress__bar ''"
  >
    <div
      class="pf-v5-c-progress__indicator"
      style="width: 42%;"
    >
      <span
        class="pf-v5-c-progress__measure"
      >
        ReactNode
      </span>
    </div>
  </div>
</DocumentFragment>
`;
